--- workspace.bzl	2019-05-24 10:14:35.890473000 -0300
+++ workspace.bzl_new	2019-05-24 17:12:28.214933424 -0300
@@ -35,7 +35,7 @@
   native.http_archive(
     name = "gemmlowp",
     url = "http://github.com/google/gemmlowp/archive/8b20dd2ce142115857220bd6a35e8a081b3e0829.tar.gz",
-    sha256 = "9cf5f1e3d64b3632dbae5c65efb79f4374ca9ac362d788fc61e086af937ff6d7",
+    sha256 = "4fbbf7b8c41dd9d2d2500a594bcda70a5be7b64d607c1556b1be9737b2396e16",
     strip_prefix = "gemmlowp-8b20dd2ce142115857220bd6a35e8a081b3e0829",
   )
 
@@ -61,7 +61,7 @@
 
   native.new_http_archive(
     name = "jpeg_archive",
-    url = "http://www.ijg.org/files/jpegsrc.v9a.tar.gz",
+    url = "https://download.videolan.org/contrib/jpeg/jpegsrc.v9a.tar.gz",
     sha256 = "3a753ea48d917945dd54a2d97de388aa06ca2eb1066cbfdc6652036349fe05a7",
     strip_prefix = "jpeg-9a",
     build_file = str(Label("//:jpeg.BUILD")),
@@ -85,7 +85,7 @@
 
   native.new_http_archive(
     name = "six_archive",
-    url = "http://pypi.python.org/packages/source/s/six/six-1.10.0.tar.gz",
+    url = "https://pypi.python.org/packages/source/s/six/six-1.10.0.tar.gz",
     sha256 = "105f8d68616f8248e24bf0e9372ef04d3cc10104f1980f54d57b2ce73a5ad56a",
     strip_prefix = "six-1.10.0",
     build_file = str(Label("//:six.BUILD")),
@@ -99,7 +99,7 @@
   native.http_archive(
     name = "protobuf",
     url = "http://github.com/google/protobuf/archive/v3.1.0.tar.gz",
-    sha256 = "0a0ae63cbffc274efb573bdde9a253e3f32e458c41261df51c5dbc5ad541e8f7",
+    sha256 = "fb2a314f4be897491bb2446697be693d489af645cb0e165a85e7e64e07eb134d",
     strip_prefix = "protobuf-3.1.0",
   )
 
@@ -141,7 +141,7 @@
   native.new_http_archive(
     name = "grpc",
     url = "http://github.com/grpc/grpc/archive/d7ff4ff40071d2b486a052183e3e9f9382afb745.tar.gz",
-    sha256 = "a15f352436ab92c521b1ac11e729e155ace38d0856380cf25048c5d1d9ba8e31",
+    sha256 = "81a20a2462cac1df00caa5ea9c05ced4633d657b19ed1ec3c6cf6e4f2bdbfce9",
     strip_prefix = "grpc-d7ff4ff40071d2b486a052183e3e9f9382afb745",
     build_file = str(Label("//:grpc.BUILD")),
   )
@@ -201,9 +201,9 @@
 
   native.new_http_archive(
     name = "avro_archive",
-    url = "http://www-us.apache.org/dist/avro/avro-1.8.0/cpp/avro-cpp-1.8.0.tar.gz",
-    sha256 = "ec6e2ec957e95ca07f70cc25f02f5c416f47cb27bd987a6ec770dcbe72527368",
-    strip_prefix = "avro-cpp-1.8.0",
+    url = "http://www-us.apache.org/dist/avro/avro-1.9.0/cpp/avro-cpp-1.9.0.tar.gz",
+    sha256 = "3a66aa600dbb171cda4664fce1c5a73a9720c8b0f55b13f76650453be6d6ab97",
+    strip_prefix = "avro-cpp-1.9.0",
     build_file = str(Label("//:avro.BUILD")),
   )
 
@@ -217,7 +217,7 @@
 
   native.new_http_archive(
     name = "bzip2_archive",
-    url = "http://www.bzip.org/1.0.6/bzip2-1.0.6.tar.gz",
+    url = "https://ufpr.dl.sourceforge.net/project/bzip2/bzip2-1.0.6.tar.gz",
     sha256 = "a2848f34fcd5d6cf47def00461fcb528a0484d8edef8208d6d2e2909dc61d9cd",
     strip_prefix = "bzip2-1.0.6",
     build_file = str(Label("//:bzip2.BUILD")),
@@ -225,9 +225,9 @@
 
   native.new_http_archive(
     name = "zlib_archive",
-    url = "http://zlib.net/zlib-1.2.8.tar.gz",
-    sha256 = "36658cb768a54c1d4dec43c3116c27ed893e88b02ecfcb44f2166f9c0b7f2a0d",
-    strip_prefix = "zlib-1.2.8",
+    url = "http://zlib.net/zlib-1.2.11.tar.gz",
+    sha256 = "c3e5e9fdd5004dcb542feda5ee4f0ff0744628baf8ed2dd5d66f8ca1197cb1a1",
+    strip_prefix = "zlib-1.2.11",
     build_file = str(Label("//:zlib.BUILD")),
   )
 
